e2e/cnab_test.go

```diff
@@ -50,7 +50,7 @@ func TestCallCustomStatusAction(t *testing.T) {
 			icmd.RunCmd(cmd).Assert(t, icmd.Success)
 
 			// docker app install
-			cmd.Command = dockerCli.Command("app", "install", path.Join(testDir, "bundle.json"), "--name", testCase.name)
+			cmd.Command = dockerCli.Command("app", "run", path.Join(testDir, "bundle.json"), "--name", testCase.name)
```

Comment:
I think this shouldn't work anymore right? `docker app run` should only run app image reference or id?

Reply:
We may have to manually put those bundles in the bundle store, or push/pull those bundles?

Reply:
Aside this renaming we discussed introducing `--cnab-bundle` option for this specific scenario.
If you don't mind, we could keep this in place and revisit this testcase and related codebase once we work on this topic.